packages/atomic/src/components/atomic-result-list/atomic-result-list.pcss

```diff
@@ -0,0 +1,7 @@
+atomic-result + atomic-result::before {
```

Comment:
Prior to this PR, Atomic did not add dividers but the default template added one.

In this PR, I moved the divider to the result list.

Reply:
Can you add a note in this css file: careful, these styles aren't protected by the shadow Dom